_includes/events.ejs

```diff
@@ -0,0 +1,40 @@
+---
+layout: base-layout.ejs
+pagination:

```

Comment:
I don't think pagination is required here... We would list all the events on the same page they are no more than 25 (though they are only 8-10). Its completely fine to list them all.